tests/xbmcgui.py

```diff
@@ -230,12 +230,12 @@ class WindowXML(Window):
 
     def __init__(self):
         ''' A stub constructor for the xbmcgui WindowXML class '''
-        super(WindowXML, self).__init__()
+        super(WindowXML, self).__init__()  # pylint: disable=super-with-arguments
```

Comment:
This is a hard one as pylint for py27 is conflicting with pylint for py3.
If we cannot fix that globally, this may be necessary. I would need to check what we do elsewhere.

Reply:
We've also whitelisted this on our other add-ons.

Reply:
Ok, this is wrong. Since this is in xbmcgui.py we don't want to do this in .pylintrc, sorry for the mixup.